deploy/examples/csi/nfs/storageclass.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
missing `---` as start of the file?

Reply:
> missing `---` as start of the file?

none of the other yamls have it, I think it is fine not to have it since we have only one document in each yaml and its optional.